### PR TITLE
aspell, hunspell, ispell: add Russian translation

### PR DESCRIPTION
**Pull request by trolzen**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages.ru/common/hunspell.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+> Проверка орфографии.
+> Больше информации: <https://hunspell.github.io/>.
+
+- Проверить орфографию в указанном файле:
+
+`hunspell {{путь/до/файла}}`
+
+- Проверить орфографию в указанном файле, используя американский словарь (en_US):
+
+`hunspell -d {{en_US}} {{путь/до/файла}}`
```
Comment (EmilySeville7cfg) [inline pages.ru/common/hunspell.md:12]: Just note that we prefer long options. Maybe the original page should be updated if the corresponding long option is available. :) The same applies to almost all other code samples.

**Review by EmilySeville7cfg — APPROVED** [review]
This PR should be checked by this command users.

File: pages.ru/linux/ispell.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+`ispell`
+
+- Проверить на ошибки указанный файл и интерактивно применить исправления:
+
+`ispell {{путь/до/файла}}`
+
+- Отобразить версию:
+
+`ispell -v`
```
Comment (EmilySeville7cfg) [inline pages.ru/linux/ispell.md:16]: Long option should be used.

File: pages.ru/common/hunspell.md (hunk 1 above)
Comment (kbdharun) [reply pages.ru/common/hunspell.md:12]: I remember using this command before. Long arguments aren't present in it. I will check it once again. Yep, it only has short arguments. [Ref](https://github.com/hunspell/hunspell)

File: pages.ru/linux/ispell.md (hunk 2 above)
Comment (kbdharun) [reply pages.ru/linux/ispell.md:16]: This command too doesn't support longer arguments, The link can be updated to a more specific one (https://www.cs.hmc.edu/~geoff/ispell-man.html) I will do it in a new PR.

**Review by kbdharun — APPROVED** [review]
Thanks for your contribution. I will merge this PR now, I will create a new one to update links as I mentioned above.

**Pull request merged**